Example/MatrioskaExample/TileViewController.swift

```diff
@@ -9,7 +9,7 @@
 import UIKit
 import Matrioska
 
-struct TileConfig: MaterializableComponentMeta {
+struct TileConfig: ExpressibleByComponentMeta {
```

Comment:
Done (wanted to see how it works), I created a runtastic team on buddybuild. Will invite you guys. Btw I think buddybuild also supports libraries now?